hail/src/main/scala/is/hail/linalg/LAPACK.scala

```diff
@@ -20,19 +21,25 @@ class UnderscoreFunctionMapper extends FunctionMapper {
 object LAPACK {
   private[this] val libraryInstance = ThreadLocal.withInitial(new Supplier[LAPACKLibrary]() {
     def get() = {
-      val standard = Native.loadLibrary("lapack", classOf[LAPACKLibrary]).asInstanceOf[LAPACKLibrary]
+      val library_name = if (HailContext.isInitialized && HailContext.getFlag("mkl_path") != null) {
+        NativeLibrary.addSearchPath("mkl_rt", "/Users/pschulz/opt/miniconda3/envs/hail/lib")
```

Comment:
You've hardcoded a path to your own laptop here.

Reply:
Whoops, thanks. I had that using the environment variable, then I lost that code and had to quickly reconstruct it.